package.json

```diff
@@ -16,7 +16,8 @@
     "lint": "cross-env NODE_OPTIONS=--no-deprecation next lint",
     "lint:fix": "cross-env NODE_OPTIONS=--no-deprecation next lint --fix",
     "check": "cross-env NODE_OPTIONS=--no-deprecation next lint && tsc --noEmit",
-    "test": "cross-env NODE_OPTIONS=--no-deprecation echo 'No tests implemented yet'",
+    "test": "cross-env NODE_OPTIONS=--no-deprecation jest",
```

Comment:
should be run during build in pipeline. After build but with the container service running to have a database.